Solutions/Windows Server DNS/Hunting Queries/DNS_CommonlyAbusedTLDs.yaml

```diff
@@ -26,7 +25,6 @@ query: |
   | where tld in~ (abusedTLD)
   | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), NameCount = count() by Name, ClientIP, tld
   | order by NameCount desc
-  | extend timestamp = StartTime, IPCustomEntity = ClientIP, DomainCustomEntity = Name
```

Comment:
We should require to add new mapping in Hunting query to ensure the mapping happens when query is deployed.

Reply:
thanks @devikamehra new method incorporated.